Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -403,7 +351,21 @@ class CombinatorialKalmanFilter {
       }
 
       if (!result.filtered &&
-          filterTargetReached(state, stepper, navigator, logger())) {
+          targetReached(state, stepper, navigator, logger())) {
```

Comment:
What does this target reached look for now? Filter target I suppose?

Reply:
right now I remember - I though of the case where we run the CKF inwards which needs to terminate at the perigee in case of a cylindrical detector